pkg/mimirtool/client/client.go

```diff
@@ -99,12 +101,13 @@ func New(cfg Config) (*MimirClient, error) {
 		Client:    client,
 		apiPath:   path,
 		authToken: cfg.AuthToken,
+		apiPrefix: cfg.APIPrefix,
 	}, nil
 }
 
 // Query executes a PromQL query against the Mimir cluster.
 func (r *MimirClient) Query(ctx context.Context, query string) (*http.Response, error) {
```

Comment:
This method is used in the `alerts` command too. Can you expose `-api-prefix` in that command too? Otherwise it will be unset and the full URL will be `ADDRESS/api/v1/query` instead of `ADDRESS/prometheus/v1/query` like it is now.